src/VirtualList/VirtualList.js

```diff
@@ -7,22 +7,36 @@
 import DOMUtil from '../Util/DOMUtil';
 import React from 'react';
 import ReactDOM from 'react-dom';
+import throttle from 'lodash.throttle';
+
+import BindMixin from '../Mixin/BindMixin';
 import Util from '../Util/Util';
 
 let mathMax = Math.max;
 let mathMin = Math.min;
 let mathFloor = Math.floor;
 let mathCeil = Math.ceil;
 
-class VirtualList extends React.Component {
+class VirtualList extends Util.mixin(BindMixin) {
+  get methodsToBind() {
+    return [
+      'onScroll',
+      'getBox',
+      'getItems',
+      'getItemsToRender',
+      'getVirtualState',
+      'visibleItems'
+    ]
+  }
+
   constructor() {
     super(...arguments);
 
     this.state = this.getVirtualState(this.props);
 
     // Replace onScroll by throttling
     if (this.props.scrollDelay > 0) {
-      this.onScroll = Util.throttle(
+      this.onScroll = throttle(
         this.onScroll.bind(this),
```

Comment:
Why do we need to bind `onScroll` again? Shouldn't is be bound to the correct scope by utilizing the bind mixin?